src/rust/src/sapling.rs

```diff
@@ -210,26 +213,26 @@ impl BundleAssembler {
         zkproof: [u8; 192], // GROTH_PROOF_SIZE
     ) -> bool {
         // Deserialize the value commitment
-        let cv = match de_ct(jubjub::ExtendedPoint::from_bytes(cv)) {
+        let cv = match Option::from(ValueCommitment::from_bytes_not_small_order(cv)) {
```

Comment:
`cv` should already have been checked by consensus to be not-small-order, so the redundant check here should not matter.